Frontend/components/categories/categoryTable.js

```diff
@@ -41,6 +41,7 @@ const CategoryTable = () => {
                                 <TableCell>Name</TableCell>
                                 <TableCell>Description</TableCell>
                                 <TableCell>Product ID</TableCell>
+                                <TableCell></TableCell>
```

Comment:
Why is it empty? :)

Reply:
because of the line above the delete icon. there was non line there and it looked not really nice if there is empty

Reply:
Could you do a screenshot?

Reply:
<img width="715" alt="image" src="https://user-images.githubusercontent.com/89083753/174137133-1c7fb19e-5d6e-4a53-ab0b-963cb366637d.png">
here :)